bin/release

```diff
@@ -1,2 +1,5 @@
 #!/bin/sh
-echo "--- {}"
+cat << EOF
+---
+default_process_types:
+EOF
```

Comment:
what does this do?

Reply:
It's an optional file for buildpacks that sets runtime defaults. In this case, we are not overriding any defaults. I was experimenting with it but was ultimately unnecessary. I'm on the fence about keeping it. It's nice to be explicit, but it's also not doing anything. 

https://devcenter.heroku.com/articles/buildpack-api#bin-release